README.md

```diff
@@ -48,6 +48,17 @@ c.BaseURL = "https://your.host.name/v1"
 ...
 ```
 
+Use customized proxy settings In case you need to use transport other than ``htttp.DefaultTransport``:
```

Comment:
You don't need two backticks (\`) around http.DefaultTransport. Just one. Also you have an extra 't' in http.

\`http.DefaultTransport\`